cmd/awscollector/Dockerfile

```diff
@@ -52,6 +52,7 @@ COPY --from=build /workspace/config/eks/prometheus/config-all.yaml /etc/eks/prom
 COPY --from=build /workspace/config/ecs/container-insights/otel-task-metrics-config.yaml /etc/ecs/container-insights/otel-task-metrics-config.yaml
 COPY --from=build /workspace/config/ecs/ecs-default-config.yaml /etc/ecs/ecs-default-config.yaml
 COPY --from=build /workspace/config/ecs/otel-instance-metrics-config.yaml /etc/ecs/otel-instance-metrics-config.yaml
+COPY --from=build /workspace/config/apprunner/apprunner-xray-config.yaml /etc/ecs/apprunner-xray-config.yaml
```

Comment:
Recommend to rephrase. /etc/ecs/apprunner-xray-config.yaml to /etc/apprunner/apprunner-xray-config.yaml

Reply:
updated